learners/setup.md

```diff
@@ -43,8 +43,8 @@ Launch **DB Browser for SQLite** to confirm that the installation was successful
 ### Am I ready?
 
 The [first lesson episode](episodes/00-sql-introduction.md) has instructions
-on loading the data in DB Browser. If you can follow the instructions under "Relational
-Databases" or "Import", everything should have downloaded and installed correctly.
+on loading the data in DB Browser. To test your setup, you can follow the instructions for importing your data into DB Browser under the sections [Relational
+Databases](https://datacarpentry.org/sql-ecology-lesson/00-sql-introduction.html#relational-databases) or [Import](https://datacarpentry.org/sql-ecology-lesson/00-sql-introduction.html#import).
```

Comment:
This will protect these links from breaking in the future if the URL changes AND it will allow someone viewing this file directly on GitHub to have a link to it

```suggestion
Databases](../episodes/00-sql-introduction.md#relational-databases) or [Import](../episodes/00-sql-introduction.md#import).
```

Reply:
Good to know. Thanks!